apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityAdminSettings.tsx

```diff
@@ -71,13 +76,24 @@ const MunicipalityAdminSettings = ({ currentMunicipality }: Props) => {
     return false
   }
 
+  const errorCheckDesemberAid = () => {
+    if (state.decemberCompensation === 0) {
```

Comment:
Shouldn't this be a postive amount as well? Or is -100 legit?

Reply:
the number input got this covered ;)